2020-07-23 15:14:06,351 INFO 36372 [egg-logrotator] agent logger reload: got log-reload message
2020-07-23 21:04:45,168 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 2587943ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-23 21:10:04,982 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 258422ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-23 22:09:18,427 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 572882ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 00:18:54,908 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 7216333ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 02:20:18,636 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 7230463ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 04:21:02,502 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 7222983ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 08:21:23,491 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 14420319ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 08:38:28,730 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 1004654ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

2020-07-24 10:37:32,488 ERROR 36372 nodejs.ClusterClientNoResponseError: client no response in 2102871ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/yqg/personal/projects/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 36372
hostname: yqgdeMBP-10.lan

